### DISCUSSION TITLE
@types/fabric

### DISCUSSION THREAD

**Discussion by peacefingers** [DefinitelyTyped/DefinitelyTyped#58213] [Issues with a @types package] [answered]
for `@types/fabric`

fabric.Image can be initialized with `HTMLCanvasElement` according to 
http://fabricjs.com/docs/fabric.Image.html#initialize
but types package only accepts `HTMLImageElement | HTMLVideoElement | String`

[a comment by an automated account is not shown]

**Comment by martin-badin** [comment, marked as answer]
Hi, @peacefingers. I created PR to fix this issue #58215

**Reply by peacefingers** [reply]
wonderful
